src/chimera/interfaces/focuser.py

```diff
@@ -26,7 +26,20 @@
 
 FocuserFeature = Enum("TEMPERATURE_COMPENSATION",
                       "ENCODER",
-                      "POSITION_FEEDBACK")
+                      "POSITION_FEEDBACK",
+                      "CONTROLLABLE_X",
+                      "CONTROLLABLE_Y",
+                      "CONTROLLABLE_U",
+                      "CONTROLLABLE_V",
+                      "CONTROLLABLE_W")
+
+FocuserAxis = Enum("X", "Y", "Z", "U", "V", "W")
+
+ControllableAxis = {FocuserFeature.CONTROLLABLE_X: FocuserAxis.X,
+                    FocuserFeature.CONTROLLABLE_Y: FocuserAxis.Y,
+                    FocuserFeature.CONTROLLABLE_U: FocuserAxis.U,
+                    FocuserFeature.CONTROLLABLE_V: FocuserAxis.V,
+                    FocuserFeature.CONTROLLABLE_W: FocuserAxis.W}
```

Comment:
Since we are already messing with the backwards compatibility I don't think that it is a bad idea to generalize a focuser that does not moves on Z axis and include this.